sdcm/cluster_aws.py

```diff
@@ -276,7 +277,8 @@ def __init__(self, ec2_instance, ec2_service, credentials,
 
             self._ec2_service.create_tags(Resources=[self._instance.id],
                                           Tags=tags_list)
-
+        if self._instance.spot_instance_request_id and 'spot' in self._instance.instance_lifecycle.lower():
```

Comment:
I think it is better to put this in `start_task_threads` as you did in `stop_task_threads`

Reply:
Done